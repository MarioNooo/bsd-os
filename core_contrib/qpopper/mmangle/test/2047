Date: Thu, 2 Jan 1992 12:21:18 -0800 (PST)
From: =?iso-8859-1?Q?=60=61=20Laurence=20Lundblade?= <lgl@cac.washington.edu>
Sender: Laurence Lundblade <lgl@cac.washington.edu>
Reply-To: Laurence Lundblade <lgl@cac.washington.edu>
Subject: =?isoxx?Q?=C0=C1=C2=C3=C4=C5=C6=C7=C8=C9=CA=CB=CC=CD=CE=CF?= xxy
  =?isofoo?B?B64SUCKS?=
To: =?iso-8859-1?Q?Keld_J=F8rn_Simonsen?= <keld>,
  =?iso-8859-1?Q?Olle_J=E4arnefors?=
yyy: extra header
xx: extra header
MIME-Version: 1.0
Content-type: text/plain;
Content-transfer-encoding: quoted-printable

This=20has=20spaces=20and=
 is one line.

