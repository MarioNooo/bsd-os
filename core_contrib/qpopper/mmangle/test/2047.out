Date: Thu, 2 Jan 1992 12:21:18 -0800 (PST)
From: `a Laurence Lundblade <lgl@cac.washington.edu>
Subject: ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏ xxy
  B64SUCKS
To: Keld Jørn Simonsen <keld>,
  Olle Jäarnefors

This has spaces and is one line.

